timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -0,0 +1,198 @@
+# Win your NFL fantasy league with TimescaleDB
+
+This tutorial is a step-by-step guide on how to ingest and analyze american football data with TimescaleDB.
+
+The dataset that we're using is provided by the National Football League (NFL) and contains data about 
+all the passing plays of the 2018 NFL season. We're going to ingest this dataset with Python into TimescaleDB 
+and start exploring it to discover interesting things about players that could help you win your next fantasy season. 
+If you aren't an NFL fan, don't worry, you will still find this tutorial helpful as it's showcasing how to 
+get started with TimescaleDB and explore a real world dataset with SQL and Python.
+
+
+1. [Create tables][]
+2. [Ingest data from CSV files][] 
+3. [Analyze NFL data][]
+   
+## Prerequisites
+
+* Python 3
+* TimescaleDB (see [installation options][install-timescale]) 
+* [Psql][psql-install] or any other PostgreSQL client (e.g. DBeaver)
+* Pandas and Matplotlib (`pip install pandas matplotlib`)
+
+## Download the dataset
+
+Download dataset here!
+
+## Create tables
+
+You will need to create four tables:
+
+* game
+  
+  Information about each game.
+* player
+  
+  Player information.
+* play
+  
+  Play information
+* tracking
+  
+  Tracking information from each play.
```

Comment:
Would be great to add a few key fields from each table or something to describe this data just a little bit more.